grails-app/conf/Config.groovy

```diff
@@ -87,13 +87,16 @@ grails.hibernate.pass.readonly = false
 // configure passing read-only to OSIV session by default, requires "singleSession = false" OSIV mode
 grails.hibernate.osiv.readonly = false
 
+quartz.autoStartup = false
+
 environments {
     development {
         grails.logging.jul.usebridge = true
     }
     production {
         grails.logging.jul.usebridge = false
-        // TODO: grails.serverURL = "http://www.changeme.com"
+        quartz.autoStartup = true
+        grails.serverURL = "http://www.mama-ng-scheduler.com"
```

Comment:
It feels like we should be setting this to something like ``http://www.example.com/`` an then setting it properly in production?